src/components/va-accordion/va-accordion.tsx

```diff
@@ -74,6 +74,21 @@ export class VaAccordion {
     }
 
     clickedItem.setAttribute('open', !prevAttr);
+
+    if (!this.isScrolledIntoView(clickedItem)) {
+      clickedItem.scrollIntoView();
```

Comment:
The default scrolling behavior uses smooth scroll, which isn't a great experience for users who might be triggered by the motion. Some users may experience dizziness or nausea. Could we do something to prevent this? Thinking of polling `window.matchMedia` ?

https://since1979.dev/respecting-prefers-reduced-motion-with-javascript-and-react/

```javascript
const mediaQuery = window.matchMedia("(prefers-reduced-motion: reduce)");

if (!this.isScrolledIntoView(clickedItem) && window.match) {
  clickedItem.scrollIntoView({ behavior: auto });
}

if (!this.isScrolledIntoView(clickedItem) && !window.match) {
  clickedItem.scrollIntoView();
}
```

Reply:
I believe the default behavior is `auto`, and not `smooth` so `clickedItem.scrollIntoView({ behavior: auto });` and `clickedItem.scrollIntoView();` are equivalent. See: https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#parameters

Reply:
@Rleahy22 Yep, I was trying to multi-task and misread the MDN docs. Please disregard my comment, and thank you for looking into it!